recipes-devtools/clang/clang/0037-clangd-Add-a-build-option-to-disable-building-dexp.patch

```diff
@@ -0,0 +1,89 @@
+From 66011b9d3b8ef4fce5d70f11a381e7cc83dff4c8 Mon Sep 17 00:00:00 2001
+From: Yoann Congal <yoann.congal@smile.fr>
+Date: Tue, 25 Mar 2025 22:25:55 +0100
+Subject: [PATCH] clangd: Add a build option to disable building dexp
+
+Upstream-Status: Pending
```

Comment:
I'll need to send this upstream and update the Upstream-Status but I wanted to make this PR available as soon as possible to fix the AB build.

Reply:
Submitted PR: https://github.com/llvm/llvm-project/pull/133124